src/Enhavo/Bundle/AppBundle/Resources/public/css/theme.css

```diff
@@ -21,81 +21,175 @@ img {max-width:100%;}
 
 .responsive-menu-btn{cursor: pointer;z-index: 1;background-image: url(../img/menu.svg);background-repeat: no-repeat;width: 30px;height: 30px;position: absolute;left: 20px;top: 20px;display: none;}
-.responsive-menu-btn:after{content:"Menu";text-transform: uppercase;color: #0c0c0c;font-size: 14px;
-    padding:2px 40px;
-    display: block;}
-
-#menu-main{
-    margin-top: 60px;
-    width: 200px;
-    height: 100%;
-}
+.responsive-menu-btn:after{content:"Menu";text-transform: uppercase;color: #072129;font-size: 14px;padding:1px 40px;display: block;letter-spacing: 0.05em;font-weight: 700;font-size: 12px;}
 
-#menu-main li a {
-    background-position: 95% 50% !important;
-    -moz-filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#grayscale");
-    -o-filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#grayscale");
-    -webkit-filter: grayscale(100%);
-    filter: gray;
-    filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#grayscale");
-    display:block;padding:10px 40px 10px 12px;background-position:156px 10px;background-repeat:no-repeat;color:#eee;text-align:right;
-}
+h1.cat{font-size: 36px;color: #454545;font-weight: lighter;padding: 30px 0px;margin-left: -15px;text-align: left;}
+
+#menu-main{width: 230px;height: 100%;}
+#menu-main .desc {position: absolute;bottom: 25px;font-size: 12px;line-height: 1.45em;color: #737677;padding: 0 0 0 15px;}
+#menu-main .desc a{color: #FFF;}
+#menu-main-container .menu-logo{width: 170px;position: relative;left: 50%;margin-left: -85px;margin-bottom: 45px;margin-top: 37px;}
+#menu-main li a {text-transform: uppercase;font-weight: 600;letter-spacing: 1px;display:block;padding: 10px 40px 10px 40px;background-position:156px 10px;background-repeat:no-repeat;color:#a8b2b4;text-align:left;font-size: 12px;}
+#menu-main li.selected{background: #20383f;}
 #menu-main li a:hover,
-#menu-main li.selected a {color:#fff;background-color:#4395d1;filter:none;}
-#menu-main li a.menu-icon-logout {background-image:url(../img/icon-logout.png);}
+#menu-main li.selected a {color:#fff;;filter:none;}
+#menu-main-container {position: fixed;background:#072129;height: 100%;}
 
-#menu-main-container {
-    position: fixed;background:#191919;height: 100%;}
+#menu-main li a:before{font-size: 16px;color:#a8b2b4;font-family: "enhavo_iconfont";font-weight: normal;padding-right: 10px;}
+.menu-icon-dashboard:before{content: '\eb4e';}

```

Comment:
We are trying to separate the bundles and the app bundle is on top level, so it should't have dependencies or references to other bundles (with some exceptions like AssetBundle). So please move the icon styles to the bundle style files again.